utils.js

```diff
@@ -141,3 +156,9 @@ function GetFailureCount(datarow) {
 function SortFailures(a, b) {
     return b["failure_count"] - a["failure_count"];
 }
+
+function SortPriority(a, b) {
+    let priorityA = a["priority"] === "--" ? 5 : parseInt(a["priority"].slice(1));
```

Comment:
If we haven't triaged it yet, arguably it should be priority 1 or 0, so we make sure to triage it first?

Reply:
I'll change this. Thanks.